monkey/agent_plugins/build_plugin.sh

```diff
@@ -50,27 +50,33 @@ plugin_options_filename="${plugin_name_lowercase}_options"
 plugin_options_filepath="${PLUGIN_PATH}/src/${plugin_options_filename}.py"
 plugin_options_model_name="${plugin_name}Options"
 
-python3.11 << EOF
+config_schema_generated=$(python3.11 << EOF
 
 import json
 from pathlib import Path
 
 def construct_config_schema():
   config_schema = {"type": "object"}
-  print("Generating config-schema.json from ${plugin_options_filepath}.")
```

Comment:
I think this was valuable debugging information. Why was it removed?

Reply:
It is not removed. It has just moved down.

Reply:
Nevermind, with the exit codes it got back where it was